packages/shared-metrics/src/libuv.js

```diff
@@ -47,5 +47,7 @@ function sense() {
     stats.statsSupported = true;
     return stats;
   }
-  return {};
+  return {
+    statsSupported: false
```

Comment:
FYI I have added consistency changes for libuv and gc to have the `statsSupported` key always available in the metrics result. Before these optimisations the key did not exist for some cases.

Especially in this line, I tried to read the actual `exports.currentPayload.statsSupported` value, which is set on failed event, but it is impossible to access it without refactoring the code.

Reply:
The decision to not transmit statsSupported = false might have been deliberate back in the days (to save a few bytes on the wire). But I agree, it is probably not worth it and having the value always present as a boolean makes sense.